docs/source/using/managing-minishift.adoc

```diff
@@ -13,11 +14,47 @@ toc::[]
 
 When you use {project}, you interact with two components:
 
-- a virtual machine (VM) created by {project}
+- the {project} virtual machine (VM)
 - the OpenShift cluster provisioned by {project} within the VM
 
-The following sections contain information about managing the {project} VM.
-For details about using {project} to manage your local OpenShift cluster, see the xref:../openshift/index.adoc#[Interacting with OpenShift] section.
+The xref:../using/managing-minishift.adoc#diagram-minishift-architecture[{project} architecture] diagram outlines these components.
+The `minishift` binary, placed somewhere on `PATH`, is used to xref:../using/managing-minishift.adoc#minishift-start-overview[`start`], xref:../using/managing-minishift.adoc#minishift-stop-overview[`stop`] and xref:../using/managing-minishift.adoc#minishift-delete-overview[`delete`] the {project} VM.
+It also interacts with the Docker daemon running within the VM, for example via xref:../using/docker-daemon.adoc#reusing-docker-daemon[`minishift docker-env`].
+Finally it also communicates with the OpenShift cluster via sub-commands of the xref:../command-ref/minishift_openshift.adoc#[`minishift openshift`] command.
+
+As {project} user you interact with the OpenShift cluster using mainly the `oc` binary.
```

Comment:
This sentence sounds awkward to me due to the "using mainly" -- maybe something along the lines of the following would flow better:

"As a {project} user you mainly interact with the OpenShift cluster using the `oc` binary."

Reply:
Sure, will fix.